CHANGELOG.md

```diff
@@ -19,8 +24,8 @@ and this project adheres to
 
 ### Fixed
 
-- cosmwasm-std: Fix `Uint64::multiply_ratio` and `Uint128::multiply_ratio` so
-  that internal multiplication cannot cause an unnecessary overflow. ([#920])
+- cosmwasm-std: Fix `Uint64::multiply_ratio` and `Uint128::multiply_ratio`
+  so that internal multiplication cannot cause an unnecessary overflow. ([#920])
```

Comment:
This "so" was moved up by my because of the text formatter (See `./devtools/format_md.sh` and #951)